components/Content/Article.jsx

```diff
@@ -1,41 +1,72 @@
+import { useMemo, useState } from 'react';
 import PropTypes from 'prop-types';
-import { useState } from 'react';
 import Image from 'next/image';
 import Link from 'next/link';
+import { isArray } from 'lodash';
+
+import { CARD_CLASS } from 'common-util/classes';
 
 const imageDomain = process.env.NEXT_PUBLIC_API_URL;
 
-const Article = ({ article }) => {
+const Article = ({
+  article, href, showReadTime, showDate,
+}) => {
   const [imageError, setImageError] = useState(false);
 
-  const { title, datePublished, slug } = article.attributes;
-  const image = article?.attributes?.headerImage?.data[0]?.attributes?.formats?.large;
+  const image = useMemo(() => {
+    const imageData = article?.attributes?.headerImage?.data;
+    const data = isArray(imageData) ? imageData?.[0] : imageData;
+
+    if (!data) return null;
+
+    return data?.attributes?.formats?.large;
```

Comment:
wonder won't it be easy just to return `article?.attributes?.headerImage?.data[0]?.attributes?.formats?.large;`
wdyt